src/components/ComeBackFeedback/ComeBackFeedback.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import SolidButton from '../SolidButton/SolidButton';
+import feedback from '../../img/feedback.svg';
+import comeBackLater from '../../img/comeBackLater.svg';
+import './ComeBackFeedback.css';
+
+const ComeBackFeedback = () => {
+    return (
+        <div className="comeback-feedback">
+
+            <div className="comeback-container">
+                <div className="comeback-text">
+                    <p>
+                        <strong>Come back later?</strong>
+                        <br/>
+                        You can come back later and your pathway will be saved! 
+                        <br/> 
+                        Alternatively, you can <a href="/quiz" rel="noreferrer noopener">start over</a> or export it.
+                    </p>
+                </div>
+
+                <img id="comeback-image" src={comeBackLater} alt="come back later"/>
+            </div>
+
+
+            <div className="feedback-container">
+                <img src={feedback} alt="feedback"/>
```

Comment:
when writing alt text, we want to write a description of what the image actually contains, instead of a vague conceptual label. instead of "feedback" i would suggest writing something like "design of a person holding a purple message box"